packages/cli/example/tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "$comment": "This file is managed by Nango, do not edit it manually.",
+    "include": [
+        "index.ts",
+        "**/*.ts"
+    ],
+    "exclude": [
+        "node_modules",
+        "dist",
+        "build"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "esnext",
+        "strict": true,
+        "esModuleInterop": true,
+        "skipLibCheck": true,
+        "forceConsistentCasingInFileNames": true,
+        "moduleResolution": "node16",
+        "allowUnusedLabels": false,
+        "allowUnreachableCode": false,
+        "exactOptionalPropertyTypes": true,
+        "noFallthroughCasesInSwitch": true,
+        "noImplicitOverride": true,
+        "noImplicitReturns": true,
+        "noPropertyAccessFromIndexSignature": true,
+        "noUncheckedIndexedAccess": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "declaration": false,
+        "sourceMap": true,
+        "composite": false,
+        "checkJs": false,
+        "noEmit": true
+    }
+}
```

Comment:
should the migration script or even maybe `nango dev` create this file if it doesn't exist

Reply:
good point, will add that in an other PR for both command 👍🏻